openshift/ci-operator/knative-images/broker-receiver/Dockerfile

```diff
@@ -40,7 +40,7 @@ RUN ./generate_jdk.sh /build/receiver/target/receiver-1.0-SNAPSHOT.jar
 
 RUN cp /build/receiver/target/receiver-1.0-SNAPSHOT.jar /app/app.jar
 
-FROM quay.io/openshift-knative/17-jdk-centos7 as running
+FROM registry.access.redhat.com/ubi8/openjdk-17:latest as running
```

Comment:
FYI, here we only need a base image without a JDK or JVM, so we can use a raw RHEL image

Reply:
(Like the one that we use for Go-based services)

Reply:
yeah, during PROD build that will be done